webhooks.py

```diff
@@ -99,7 +99,7 @@ def index():
 
     # Gather data
     try:
-        payload = loads(request.data)
+        payload = loads(request.data).decode('utf-8')
```

Comment:
Instead of

```python
payload = loads(request.data).decode('utf-8')
```

move the closing-bracket:

```python
payload = loads(request.data.decode('utf-8'))
```

I think your version tries to decode a JSON-object which is syntactically not valid. Is this line even required since `request.data` should return always a string and `json.loads` takes a string? (http://flask.pocoo.org/docs/0.12/api/#flask.Request.data)